File: linear_search/linearSearch.java
public class linearSearch
{
    
    public static void main(String[] args)
    {
    	int arr[] = {1, 20, 5, 41, 8, 9};
    	int n = arr.length;
    	int x = 1;
    	

        for (int i = 0; i < n; i++)
        {
        
            
            if (arr[i] == x)
            	System.out.println("Element found index is :"+ i );
        }
    }
} 

File: linear_search/linearSearch.py
def search(array,x):
    index=0
    for i in array:
        if(i==x):
            return index
        index+=1

    return -1


userArray=[]

user=int(input("Enter elements for array: enter negative value to end"))
while(user>-1):
    userArray.append(user)
    user=int(input("Enter elements for array: enter negative value to end"))

x=int(input("Enter elementto find"))
result=search(userArray,x)

if(result==-1):
    print("Element is unavailable")

else:
    print("Element is at index"+str(result))
